java-checks/src/main/java/org/sonar/java/checks/security/CryptographicKeySizeCheck.java

```diff
@@ -143,12 +123,9 @@ public void visitMethodInvocation(MethodInvocationTree mit) {
     }
 
     private boolean isSameVariableSymbol(MethodInvocationTree mit) {
-      Tree methodSelect = mit.methodSelect();
+      ExpressionTree methodSelect = mit.methodSelect();
       if (methodSelect.is(Tree.Kind.MEMBER_SELECT)) {
```

Comment:
this can be simplified to:
```
return methodSelect.is(Tree.Kind.MEMBER_SELECT) 
  && extractIdentifierSymbol(((MemberSelectExpressionTree) methodSelect).expression()).filter(s -> s.equals(variable)).isPresent();
```